app/assets/stylesheets/recordings.scss

```diff
@@ -106,3 +106,76 @@
     color: var(--brand-color) !important;
   }
 }
+
+#pulse-loading {
+  margin-left: 25px;
+}
+
+// Dot Pulse
+.dot-pulse {
+  position: relative;
+  left: -9999px;
+  width: 10px;
+  height: 10px;
+  border-radius: 5px;
+  background-color: #9880ff;
+  color: #9880ff;
+  box-shadow: 9999px 0 0 -5px;
+  animation: dot-pulse 1.5s infinite linear;
+  animation-delay: 0.25s;
+}
+.dot-pulse::before, .dot-pulse::after {
+  content: "";
+  display: inline-block;
+  position: absolute;
+  top: 0;
+  width: 10px;
+  height: 10px;
+  border-radius: 5px;
+  background-color: #9880ff;
+  color: #9880ff;
+}
+.dot-pulse::before {
+  box-shadow: 9984px 0 0 -5px;
+  animation: dot-pulse-before 1.5s infinite linear;
+  animation-delay: 0s;
+}
+.dot-pulse::after {
+  box-shadow: 10014px 0 0 -5px;
+  animation: dot-pulse-after 1.5s infinite linear;
+  animation-delay: 0.5s;
+}
+
+@keyframes dot-pulse-before {
+  0% {
+    box-shadow: 9984px 0 0 -5px;
+  }
+  30% {
+    box-shadow: 9984px 0 0 2px;
+  }
+  60%, 100% {
+    box-shadow: 9984px 0 0 -5px;
+  }
+}
+@keyframes dot-pulse {
+  0% {
+    box-shadow: 9999px 0 0 -5px;
+  }
+  30% {
+    box-shadow: 9999px 0 0 2px;
+  }
+  60%, 100% {
+    box-shadow: 9999px 0 0 -5px;
+  }
+}
+@keyframes dot-pulse-after {
+  0% {
+    box-shadow: 10014px 0 0 -5px;
+  }
+  30% {
+    box-shadow: 10014px 0 0 2px;
+  }
+  60%, 100% {
+    box-shadow: 10014px 0 0 -5px;
+  }
+}
```

Comment:
This is a lot for a simple loading animation - I think we should either look into a library or scrap this concept completely

Reply:
I feel a library would be "heavy" for this, especially if we just using it here.

I think it's a nice touch though, if you think it's too much we can scrap it all

Reply:
I think it's nice.
The color needs to be changed from `#9880ff` to `whitesmoke`, `silver` or even `brand-color`

Reply:
Yeah to me this is just a mess when it comes to maintaining. Libraries are easy to maintain because someone else does it for you. If we find an issue with this code, we have to manually go in and fix it, and then test it against a bunch of different scenarios

Reply:
Yeah makes sense. I'm thinking of using react-bootstrap since we already have it, and to just use this and change it's colour.
https://react-bootstrap.github.io/components/spinners/#:~:text=Copy-,Grow%20Spinner,-%2D%20grow

Besides that there isn't a small library for just this